Fixed an error in the example script

dilukose opened the pull request:
Added the endpointlocation parameter that is required when adding external endpoints to a profile with performance routing type

azurecla commented:
Hi __@kevinscharpenberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kevsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dilukose committed:
Fixed an error in the example script

Added the endpointlocation parameter that is required when adding external endpoints to a profile with performance routing type
